src/App.js

```diff
@@ -1,24 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import InputCalculator from './components/Calculator';
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/158527425-568ed010-5ac8-46fd-8c8b-44eef9493b4a.png)


- [ ] `Calculator` should be the component that gets imported into `App.js`, not `InputCalculator`.  Please kindly correct this so the project is constructed according to instructions.